packages/website/src/components/lib/has-own-property.ts

```diff
@@ -0,0 +1,6 @@
+export function hasOwnProperty<X extends object, Y extends PropertyKey>(
```

Comment:
Lol. What a great example of why TypeScript's `in` type narrowing sometimes falls short.

(not requesting changes, just griping about the world we live in)

Reply:
Yeah, as I remember, current TS version can infer type from using `in`.